mazda_2019.dbc

```diff
@@ -98,12 +145,6 @@ BO_ 258 NEW_MSG_3: 8 XXX
  SG_ NEW_SIGNAL_1 : 13|10@0- (1,0) [0|255] "" XXX
  SG_ NEW_SIGNAL_2 : 18|10@0- (1,0) [0|255] "" XXX
 
-BO_ 28 WHEEL_SPEEDS: 8 XXX
- SG_ RR : 55|16@0+ (0.01,-100) [0|65535] "" XXX
- SG_ RL : 39|16@0+ (0.01,-102) [0|65535] "" XXX
- SG_ FL : 7|16@0+ (0.01,-100) [0|65535] "" XXX
- SG_ FR : 23|16@0+ (0.01,-100) [0|65535] "" XXX
-
```

Comment:
Are the IDs really all different on this other bus?

Reply:
I didn't remove the signals from the DBC unless there was a overlap in the desired name. For example wheel speeds is on both busses, but address 352 may not have been on the new bus. 

I could simplify it at a later date.

Reply:
Hope that makes sense

Reply:
In the future, I will be intercepting another bus for longitudinal control so I will need another DBC file. I was thinking it might be better to sort the DBCs into brand files.